src/terminal/parser/OutputStateMachineEngine.cpp

```diff
@@ -744,6 +745,9 @@ bool OutputStateMachineEngine::ActionOscDispatch(const wchar_t /*wch*/,
     case OscActionCodes::SetCursorColor:
         success = _GetOscSetColor(string, color);
         break;
+    case OscActionCodes::CopyToClipboard:
+        success = _GetOscString(string, copyContent);

```

Comment:
As @nicm [called out](https://github.com/microsoft/terminal/pull/5823#issuecomment-626568429), we should make sure we're parsing this the same way it's defined in the standard.

Reply:
Sure, I've made this change.